src/sentry/api/endpoints/grouping_levels.py

```diff
@@ -115,18 +114,42 @@ def get_levels_overview(group):
 
     fields = res["data"][0]
 
+    if not fields["longest_hierarchical_hashes"]:
+        raise NotHierarchical()
+
+    materialized_hashes = {
+        hash
+        for hash, in GroupHash.objects.filter(project=group.project, group=group).values_list(
+            "hash"
+        )
+    }
+
+    current_level = max(
+        i
+        for i, hash in enumerate(fields["longest_hierarchical_hashes"])
+        if hash in materialized_hashes
+    )
```

Comment:
Is there any danger of `materialized_hashes` being empty and `max` failing here?

Reply:
yes, happened multiple times to me in development, but it should be a crash. it's invalid internal state